usaspending_api/references/management/commands/loadcfda.py

```diff
@@ -69,7 +69,7 @@ def handle(self, *args, **options):
 
                     cfda_program.save()
 
-                    self.logger.log(20, "loaded %s %s ", cfda_program.program_number, cfda_program)
+                    # self.logger.log(20, "loaded %s %s ", cfda_program.program_number, cfda_program)
```

Comment:
Intentionally removed it. When testing, it's more often that you _won't_ want it spitting out every line that it loads than you will. Until we implement the dynamic logger stuff for tests it's better to keep screen output limited.